sdk/digitaltwins/azure-digitaltwins-core/samples/notebooks/01 Patrons.ipynb

```diff
@@ -0,0 +1,603 @@
+{
```

Comment:
Could these files be renamed to remove space characters? e.g. `01_patrons.ipynb` or `01-Patrons.ipynb`

Reply:
Sure I'll do that.

Reply:
removed spaces from the notebooks